CenterForOpenScience/osf.io
# [Feature] Limit meetings to searching osfstorage  [OSF-7037]

## Purpose
Meetings files were allowed to search any storage provider to find a viable file for the download link. 


## Changes
Limit the provider to osfstorage to stop incorrect download links.


## Side effects
None known.



## Ticket
[#OSF-7037]
https://openscience.atlassian.net/browse/OSF-7037


## QA Considerations
1. Test that download links work properly for projects without addons
2. Test that download button shows up properly when an addon is added and a file in the addon has been clicked.
3. Test that download button does not shows up when an addon is added (and file clicked) but the file in osfstorage has been deleted

See the notes on the jira ticket for steps to reliably replicate this bug.
